examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,147 @@
+# We first import the modules we need. Note that the :mod:`~.gbs.points` module has most of
+# the core functionalities for this tutorial.
+
+import numpy as np
+import plotly
+from sklearn.datasets import make_blobs
+from strawberryfields.gbs import points, plot
+
+##############################################################################
+# We define a space where the GBS point process patterns are generated. This
+# space is referred to as the state space and is defined by a set of points. The
+# point process selects a subset of these points in each sample. Here we create
+# a 20 :math:`\times` 20 square grid of points.
+
+R = np.array([(i, j) for i in range(20) for j in range(20)])
+
+##############################################################################
+# The rows of R are the coordinates of the points.
+#
+# Next step is to create the kernel matrix for the points of this discrete space. We call
+# the :func:`~.kernel` function which uses the *radial basis function* (RBF) kernel defined as:
+#
+# .. math::
+#     K_{i,j} = e^{-\|\bf{r}_i-\bf{r}_j\|^2/2\sigma^2},
+#
+# where :math:`\bf{r}_i` are the coordinates of point :math:`i` and :math:`\sigma` is a kernel
+# parameter that determines the scale of the kernel.
+#
+# In the RBF kernel, points that are much further than a distance :math:`\sigma` from each other
+# lead to small entries of the kernel matrix, whereas points much closer than :math:`\sigma`
+# generate large entries. Now consider a specific point pattern in which all points
+# are close to each other, which simply means that their matrix elements have larger entries. The
+# permanent of a matrix is a sum over the product of some matrix entries. Therefore,
+# the submatrix that corresponds to those points has a large permanent and the probability of
+# observing them in a sample is larger.
+#
+# For kernel matrices that are positive-semidefinite, such as the RBF kernel, there exist efficient
+# quantum-inspired classical algorithms for permanental point process sampling
+# :cite:`jahangiri2019point`. In this tutorial we use positive-semidefinite kernels and the
+# quantum-inspired classical algorithm.
+#
+# Let's construct the RBF kernel with the parameter :math:`\sigma` set to 2.5.
+
+K = points.kernel(R, 2.5)
+
+##############################################################################
+# We generate 10 samples with an average number of 50 points per sample by calling
+# the :func:`~.points.sample` function of the :mod:`~.gbs.points` module.
+
+samples = points.sample(K, 50.0, 10)
+
+##############################################################################
+# We visualize the first sample by using the :func:`~.plot_points` function of
+# the :mod:`~.gbs.plot` module. The point patterns generated by the permanental point process
+# usually have a higher degree of clustering compared to a uniformly random pattern.
+
+Plot_1 = plot.plot_points(R, samples[0], point_size=10)
```

Comment:
```suggestion
plot_1 = plot.plot_points(R, samples[0], point_size=10)
```
^ I think it's convention not to capitalize